CataclysmWin.cbp

```diff
@@ -7,170 +7,309 @@
 		<Option compiler="gcc" />
 		<Build>
 			<Target title="Release">
-				<Option output="Cataclysm" prefix_auto="1" extension_auto="1" />
+				<Option output="CataclysmWin" prefix_auto="1" extension_auto="1" />
```

Comment:
@KA101, but part of C::B project file name `CataclysmWin.cbp` is `Win`, which states that this project for Windows users only, don't it?